### PR TITLE
make dep optional and resolve one fixme

### PR DESCRIPTION
**Pull request by klensy**

### CODE REVIEW

File: chrono-tz-build/Cargo.toml [hunk 1]
```diff
@@ -19,6 +19,6 @@ regex = ["dep:regex"]
 [dependencies]
 parse-zoneinfo = { version = "0.3" }
 regex = { default-features = false, version = "1", optional = true }
-phf = { version = "0.11", default-features = false }
+phf = { version = "0.11", default-features = false, optional = true }
```
Comment (djc) [inline chrono-tz-build/Cargo.toml:22]: Can you add a comment here that we only have this dependency in order to forward the `uncased` feature to `phf_shared`?

**Review by djc — APPROVED** [review]

File: chrono-tz-build/Cargo.toml (hunk 1 above)
Comment (klensy) [reply chrono-tz-build/Cargo.toml]: Actually replaced `phf` with `phf_shared` and added comment.

**Comment by djc** [comment]
Thanks, good idea!

**Comment by klensy** [comment]
Nano ping.

**Comment by djc** [comment]
Thanks!

**Pull request merged**